apps/docs/pages/guides/functions/unit-test.mdx

```diff
@@ -0,0 +1,132 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  id: 'unit-test',
+  title: 'Unit Tests for Edge Functions',
+  description: 'Unit Tests for Edge Functions with Deno Test',
+}
+To ensure the correctness and performance of your Edge Functions, testing is an essential step in the development process. Let's discuss the significance of testing and examining the code in the `deno-test.ts` file, which demonstrates how to test the Supabase client and the `hello-world` function.
```

Comment:
Maybe worth mentioning, in which directory the user should create the test file and how to name it.
(I think it's better to place them in `supabase/tests` and can name the file by function name followed by `-test.ts` eg: hello-world-test.ts`)